ofl/agdasima/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,6 @@
+<p>
+    Agdasima is a DocRepair font, intended to improve compliance with the ISO/IEC
+    29500 standard by providing fallback that minimizes text reflow in Office Open
+    XML documents. Agdasima is based on Big Shoulders, a condensed American Gothic
+    sans-serif font family.
+</p>
```

Comment:
@yanone no github repo? is this something we should ignore for all docrepair fonts?

Reply:
Adam says they're not open for contributions. I agree.